### DISCUSSION TITLE
In Rofi, How can I change the mode by entering a prefix?

### DISCUSSION THREAD

**Discussion by FawazBinSaleem** [davatorium/rofi#2023] [General]
I'm a huge fan of Alfred on macOS and I just recently switched to Linux and Rofi.

One thing I liked about Alfred is that I can enter a word or character followed by space and the whole list will change.

Example:

When I enter "c ", instead of listing the apps, Alfred would switch to the clipboard history.

Can this be done in Rofi?

If that did work, then I would have something like this:

* Default: Listing apps
* c: clipboard history
* w: Windows
* =: Calculator

... you get the idea

Sorry if this is a duplicate question, I can't really find the answer to this anywhere.

**Comment by DaveDavenport** [comment]
The combi mode can do something like this. 
By doing: '!{mode}'   you can filter down.
So ' !d application'  should give you applications via drun, '!w firefox'  should give you firefox windows via window mode . (given both are enabled in combi mode.)
